rust/agama-server/src/network/nm/client.rs

```diff
@@ -150,20 +146,13 @@ impl<'a> NetworkManagerClient<'a> {
                 .build()
                 .await?;
 
-            let device_name = proxy.interface().await?;
-            let device_type = NmDeviceType(proxy.device_type().await?);
-            if let Ok(device_type) = device_type.try_into() {
-                devs.push(Device {
-                    name: device_name,
-                    type_: device_type,
-                });
+            if let Ok(device) = DeviceFromProxyBuilder::new(&self.connection, &proxy)
+                .build()
+                .await
+            {
+                devs.push(device);
             } else {
-                // TODO: use a logger
-                log::warn!(
-                    "Ignoring network device '{}' (unsupported type '{}')",
-                    &device_name,
-                    &device_type
-                );
+                tracing::warn!("Ignoring network device: ");
```

Comment:
This looks like we forgot to add _some_ info about the device being ignored

Reply:
Oops, right.